lib/config/migrations/custom/fetch-release-notes-migration.ts

```diff
@@ -1,12 +1,21 @@
 import is from '@sindresorhus/is';
+import type { FetchChangeLogsOptions } from '../../types';
 import { AbstractMigration } from '../base/abstract-migration';
 
 export class FetchReleaseNotesMigration extends AbstractMigration {
+  override deprecated = true;
   override readonly propertyName = 'fetchReleaseNotes';
 
+  readonly newPropertyName = 'fetchChangeLogs';
+
   override run(value: unknown): void {
+    let newValue: FetchChangeLogsOptions | undefined;
```

Comment:
```suggestion
    let newValue: unknown;
```
I don't think we need the type here.